src/main/resources/config/liquibase/changelog/20190828140700_change_task_data_varchar.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="textType" value="varchar" dbms="postgresql, h2"/>
+    <property name="textType" value="varchar(1024)" dbms="oracle"/>
```

Comment:
I suggest to use more
Since Oracle Database 12c, you can specify the maximum size of 32767 for the VARCHAR2 data type. Oracle uses the MAX_STRING_SIZE parameter for controlling the maximum size. If the MAX_STRING_SIZE is STANDARD , then the maximum size for VARCHAR2 is 4000 bytes

Reply:
than maybe set this:
<property name="textType" value="varchar2(32767)" dbms="oracle"/>

Reply:
4000 ok